packages/custom-functions-metadata/src/generate.ts

```diff
@@ -140,31 +141,43 @@ type CustomFunctionsSchemaDimensionality = "invalid" | "scalar" | "matrix";
  * @param outputFileName - Name of the file to create (i.e functions.json)
  */
 export async function generateCustomFunctionsMetadata(
-  inputFile: string,
+  input: string | string[],
   wantConsoleOutput: boolean = false
 ): Promise<IGenerateResult> {
+  const inputFiles: string[] = Array.isArray(input) ? input : [input];
+  const functions: IFunction[] = [];
   const generateResults: IGenerateResult = {
     metadataJson: "",
     associate: [],
     errors: [],
   };
 
-  if (fs.existsSync(inputFile)) {
-    const sourceCode = fs.readFileSync(inputFile, "utf-8");
-    const parseTreeResult: IParseTreeResult = parseTree(sourceCode, inputFile);
-    parseTreeResult.extras.forEach((extra) => extra.errors.forEach((err) => generateResults.errors.push(err)));
-
-    if (generateResults.errors.length > 0) {
-      if (wantConsoleOutput) {
-        console.error("Errors in file: " + inputFile);
-        generateResults.errors.forEach((err) => console.error(err));
+  if (input && inputFiles.length > 0) {
+    inputFiles.forEach((inputFile) => {
+      if (!inputFile) {
+        throw new Error(`Input file name is empty or has a comma at the beginning or end.\n Input: ${input}`);
```

Comment:
I'm not familiar with this code, so this comment is probably a nit :),

This error seems very specific but also very useful.
Should we say that the filename is invalid, and then also say "For example, it has a comma at the beginning or end", something like that? Or are the filenames always valid except when there's a comma somewhere? In other words, that's the only error we can ever have in a filename from `inputFiles`?

Reply:
It is specific and maybe too much so (I wasn't sure myself).  With this change I'm introducing using a comma delimited list of file paths being input as a string and if the comma is at the beginning or the end it gets converted into an array of two strings with one being empty which could easily get overlooked (I know I could overlook it).  This was intended to help point that out.  Other solutions could be to automatically trim the comma, automatically skip empty strings in the new array, or make the error general and hope they figure it out be reading documentation.  Thoughts on a better way?

Reply:
Went ahead and changed the error checking to trim and ignore empty strings instead of throwing an error message (except for the existing check for no input at all).